src/pages/costModels/createCostModelWizard/index.tsx

```diff
@@ -336,6 +336,7 @@ class CostModelWizardBase extends React.Component<Props, State> {
           }}
         />
         <Modal
+          aria-label="quit-wizard-dialog"
```

Comment:
The aria labels should be localized, so it can be read by a screen reader

Reply:
Oh, I didn't know that. No problem. I will fix that.